src/styles/StyleUtils.ts

```diff
@@ -472,11 +431,10 @@ function getBadgeColorStyle(success, error, isPressed = false, isAdHoc = false)
 /**
  * Generate a style for the background color of the button, based on its current state.
  *
- * @param {String} [buttonState] - One of {'default', 'hovered', 'pressed'}
- * @param {Boolean} isMenuItem - whether this button is apart of a list
- * @returns {Object}
+ * @param buttonState - One of {'default', 'hovered', 'pressed'}
+ * @param isMenuItem - whether this button is apart of a list
  */
-function getButtonBackgroundColorStyle(buttonState = CONST.BUTTON_STATES.DEFAULT, isMenuItem = false) {
+function getButtonBackgroundColorStyle(buttonState: ValueOf<typeof CONST.BUTTON_STATES> = CONST.BUTTON_STATES.DEFAULT, isMenuItem = false): ViewStyle | CSSProperties {
```

Comment:
You can extract `ValueOf<typeof CONST.BUTTON_STATES>` to a type

Reply:
Done!